lib/lexer.js

```diff
@@ -26,7 +28,14 @@ module.exports = function Lexer (input) {
 
     // are we infinite looping? throw an error
     if (last === current) {
-      throw new Error(`Cannot parse character "${char}" at ${line}:${col}`)
+      throw new Error({
```

Comment:
I did this as the new Error still handles regular errors in a similar manor to our custom ones